src/Generators/ServiceGenerator.php

```diff
@@ -36,6 +36,10 @@ public function generate(): void
             }
         }
 
+        if (!$this->isStubExists($stub)) {
```

Comment:
@pirs1337 all good here, but could you please remove logic above with the stub `service_with_trait` as a separate PR